[2025-03-13T20:06:23.303+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-13T20:06:23.311+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: parse_FT.get_daily_listing_task manual__2025-03-13T20:06:19.714312+00:00 [queued]>
[2025-03-13T20:06:23.316+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: parse_FT.get_daily_listing_task manual__2025-03-13T20:06:19.714312+00:00 [queued]>
[2025-03-13T20:06:23.316+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-13T20:06:23.323+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): get_daily_listing_task> on 2025-03-13 20:06:19.714312+00:00
[2025-03-13T20:06:23.328+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=82) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-13T20:06:23.329+0000] {standard_task_runner.py:72} INFO - Started process 84 to run task
[2025-03-13T20:06:23.329+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'parse_FT', 'get_daily_listing_task', 'manual__2025-03-13T20:06:19.714312+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/scraping.py', '--cfg-path', '/tmp/tmphyk9hw6s']
[2025-03-13T20:06:23.329+0000] {standard_task_runner.py:105} INFO - Job 61: Subtask get_daily_listing_task
[2025-03-13T20:06:23.352+0000] {task_command.py:467} INFO - Running <TaskInstance: parse_FT.get_daily_listing_task manual__2025-03-13T20:06:19.714312+00:00 [running]> on host 50523e7a7f31
[2025-03-13T20:06:23.397+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='joel' AIRFLOW_CTX_DAG_ID='parse_FT' AIRFLOW_CTX_TASK_ID='get_daily_listing_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-13T20:06:19.714312+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-13T20:06:19.714312+00:00'
[2025-03-13T20:06:23.398+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-13T20:06:23.398+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-13T20:06:23.398+0000] {logging_mixin.py:190} INFO - Current task name:get_daily_listing_task state:running start_date:2025-03-13 20:06:23.311996+00:00
[2025-03-13T20:06:23.399+0000] {logging_mixin.py:190} INFO - Dag name:parse_FT and current dag run status:running
[2025-03-13T20:06:23.399+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-13T20:06:23.538+0000] {warnings.py:112} WARNING - /opt/***/dags/scraping.py:183: GuessedAtParserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("lxml"). This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.

The code that caused this warning is on line 183 of the file /opt/***/dags/scraping.py. To get rid of this warning, pass the additional argument 'features="lxml"' to the BeautifulSoup constructor.

  soup = BeautifulSoup(r.text)

[2025-03-13T20:06:36.405+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-13T20:06:36.411+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-13T20:06:36.411+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=parse_FT, task_id=get_daily_listing_task, run_id=manual__2025-03-13T20:06:19.714312+00:00, execution_date=20250313T200619, start_date=20250313T200623, end_date=20250313T200636
[2025-03-13T20:06:36.434+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-13T20:06:36.435+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-13T20:06:36.435+0000] {logging_mixin.py:190} INFO - Dag name:parse_FT queued_at:2025-03-13 20:06:19.727845+00:00
[2025-03-13T20:06:36.435+0000] {logging_mixin.py:190} INFO - Task hostname:50523e7a7f31 operator:PythonOperator
[2025-03-13T20:06:36.467+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-13T20:06:36.482+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-13T20:06:36.483+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
